runway/module/__init__.py

```diff
@@ -98,15 +100,30 @@ class RunwayModule(object):
 
     def __init__(self, context, path, options=None):
         """Initialize base class."""
+        self.name = os.path.basename(path)
+
         self.context = context
 
+        # it would be good to remove the need for sub-classes to refer
+        #  to this directly, and have them rely on 'folder' instead
         self.path = path
 
+        self.folder = RunwayModuleFolder(path)
+
         if options is None:
             self.options = {}
         else:
             self.options = options
 
+        self.environment = better_dict_get(self.options, 'environments', {}).get(context.env_name)
```

Comment:
This logic was repeated in each module... I think this is the behaviour we want for all of them.